packages/components/src/templates/next/components/complex/InfoCols/InfoCols.tsx

```diff
@@ -24,40 +21,35 @@ const InfoBoxIcon = ({ icon }: { icon?: SupportedIconName }) => {
   if (!icon) return null
   const Icon = SUPPORTED_ICONS_MAP[icon]
   return (
-    <div className="rounded-lg bg-site-primary-100 p-2">
-      <Icon className="h-auto w-6 text-site-primary" />
+    <div>
+      <Icon className="h-auto w-6 text-base-content-strong group-hover:text-brand-interaction" />
     </div>
   )
 }
 
 const InfoBoxes = ({
   infoBoxes,
-  LinkComponent,
 }: Pick<InfoColsProps, "infoBoxes" | "LinkComponent">) => {
   return (
-    <div className="grid grid-cols-1 gap-x-28 gap-y-20 md:grid-cols-2 xl:grid-cols-3">
+    <div className="grid grid-cols-1 gap-x-16 gap-y-10 md:grid-cols-2 md:gap-y-12 lg:grid-cols-3">
       {infoBoxes.map((infoBox, idx) => (
-        <div key={idx} className="flex flex-col items-start gap-5 text-left">
+        <a
+          href={infoBox.buttonUrl}
+          key={idx}
+          className="group flex flex-col items-start gap-3 text-left"
+        >
           <InfoBoxIcon icon={infoBox.icon} />
```

Comment:
nit: i usually like to add `aria-hidden` to icons since they do not provide any value to screen readers.

Reply:
addressed in https://github.com/opengovsg/isomer/pull/432/commits/d4e69116ea877800cd454f363901ae46c095ee17